packages/patterns/src/types.js

```diff
@@ -456,36 +456,67 @@ export {};
  * Matches any Passable that is matched by `subPatt` or is the exact value `undefined`.
  */
 
+/**
+ * @typedef {'never' | 'passable' | 'raw'} DefaultGuardType
+ */
+
+/**
+ * @typedef {<M extends Record<PropertyKey, MethodGuard>>(
+ *   interfaceName: string,
+ *   methodGuards: M,
+ *   options: {defaultGuards?: 'never', sloppy?: false }) => InterfaceGuard<M>
+ * } MakeInterfaceGuardStrict
+ */
 /**
  * @typedef {(
  *   interfaceName: string,
  *   methodGuards: any,
- *   options: {sloppy: true}) => InterfaceGuard<Record<PropertyKey, MethodGuard>>
+ *   options: {defaultGuards?: 'passable' | 'raw', sloppy?: true }) => InterfaceGuard<any>
  * } MakeInterfaceGuardSloppy
  */
 /**
  * @typedef {<M extends Record<PropertyKey, MethodGuard>>(
  *   interfaceName: string,
  *   methodGuards: M,
- *   options?: {sloppy?: boolean}) => InterfaceGuard<M>
+ *   options?: {defaultGuards?: DefaultGuardType, sloppy?: boolean}) => InterfaceGuard<M>
  * } MakeInterfaceGuardGeneral
  */
-/** @typedef {MakeInterfaceGuardSloppy & MakeInterfaceGuardGeneral} MakeInterfaceGuard */
+/** @typedef {MakeInterfaceGuardStrict & MakeInterfaceGuardSloppy & MakeInterfaceGuardGeneral} MakeInterfaceGuard */
 
 /**
  * @typedef {object} GuardMakers
  *
  * @property {MakeInterfaceGuard} interface
  * Guard the interface of an exo object
  *
- * @property {(...argPatterns: Pattern[]) => MethodGuardMaker0} call
- * Guard a synchronous call
+ * @property {(...argPatterns: SyncValueGuard[]) => MethodGuardMaker0} call
+ * Guard a synchronous call.  Arguments not guarded by `M.raw()` are
+ * automatically hardened and must be at least Passable.
  *
  * @property {(...argGuards: ArgGuard[]) => MethodGuardMaker0} callWhen
- * Guard an async call
+ * Guard an async call.  Arguments not guarded by `M.raw()` are automatically
+ * hardened and must be at least Passable.
  *
  * @property {(argPattern: Pattern) => AwaitArgGuard} await
- * Guard an await
+ * In parameter position, guard a parameter by awaiting it. Can only be used in
+ * parameter position of an `M.callWhen`.
+ * `M.await(M.nat())`, for example, with `await` the corresponding argument,
+ * check that the fulfillment of the `await` satisfies the `M.nat()`
+ * pattern, and only then proceed to call the raw method with that fulfillment.
+ * If the argument already passes the `M.nat()` pattern, then the result of
+ * `await`ing it will still pass, and the `M.callWhen` will still delay the
+ * raw method call to a future turn.
+ * If the argument is a promise that rejects rather than fulfills, or if its
+ * fulfillment does not satisfy the nested pattern, then the call is rejected
+ * without ever calling the raw method.
```

Comment:
:+1: :+1: for adding this.
```suggestion
 * Guard a positional parameter in `M.callWhen`, awaiting it and matching its
 * fulfillment against the provided pattern.
 * For example, `M.await(M.nat())` will await the first argument, check that its
 * fulfillment satisfies `M.nat()`, and only then call the guarded method with
 * that fulfillment. If the argument is a non-promise value that already
 * satisfies `M.nat()`, then the result of `await`ing it will still pass, and
 * `M.callWhen` will still delay the guarded method call to a future turn.
 * If the argument is a promise that rejects rather than fulfills, or if its
 * fulfillment does not satisfy the nested pattern, then the call is rejected
 * without ever invoking the guarded method.
```